src/generic/DirectSum.jl

```diff
@@ -224,22 +219,22 @@ function DirectSum(vals::AbstractAlgebra.FPModule{T}...) where T <: RingElement
    return DirectSum([vals...])
 end
 
-function hom(D::DirectSumModule{T}, A::AbstractAlgebra.FPModule{T}, m::Vector{<:ModuleHomomorphism{T}}) where T <: RingElement
+function ModuleHomomorphism(D::DirectSumModule{T}, A::AbstractAlgebra.FPModule{T}, m::Vector{<:ModuleHomomorphism{T}}) where T <: RingElement
   S = summands(D)
   length(S) == length(m) || error("map array has wrong length")
-  all(x->domain(m[i]) == S[i] && codomain(m[i]) == A, 1:length(S)) || 
+  all(i->domain(m[i]) == S[i] && codomain(m[i]) == A, 1:length(S)) || 
                            error("modules and maps are not compatible")
-  return hom(D, A, vcat([x.matrix for x = m]))
+  return ModuleHomomorphism(D, A, vcat([x.matrix for x = m]...))
 end
 
-function hom(D::DirectSumModule{T}, A::DirectSumModule{T}, m::Array{<:Any, 2}) where T <: RingElement
+function ModuleHomomorphism(D::DirectSumModule{T}, A::DirectSumModule{T}, m::Array{<:Any, 2}) where T <: RingElement
   SD = summands(D)
   SA = summands(A)
   size(m) == (length(SD), length(SA)) || error("dimensions do not match")
-  for i=1:length(SD)
-    for j=1:length(SA)
+  for i = 1:length(SD)
+    for j = 1:length(SA)
```

Comment:
Are you using an indent of 3 here like the rest of the file? Perhaps it is just GitHub making it look like two, or perhaps your editor is set up for 2 instead of whatever the file is using.